### PR TITLE
Fixing sidebar

### LINKED ISSUE #3096: Fix the Sidebar
**Actual Behaviour**
The sidebar should be fixed as when the main content of the page is large then to navigate using sidebar the user has to scroll to the top and then find the respective link.

**Expected Behaviour**
The sidebar should be fixed and made scrollable.

**Screenshots of the issue**
image

**Would you like to work on the issue?**
Yes

I am new to this project so if you can tell me the files to work then I can work on this.

### PR DESCRIPTION
**Pull request by elit-altum**
Fixes #3096 

Changes: The CSS in JS was changed to fix the navigation toggler on the bottom of the screen so the user does not have to scroll all the way up to access it.

Demo Link: https://pr-3464-fossasia-susi-web-chat.surge.sh

Screenshots of the change:
__Before__

File-animation-before

__After__

File-animation

### CODE REVIEW

**Comment by elit-altum** [comment]
> 
> 
> Make it circular similar to the other buttons on the bottom

@akshatnitd  done!

Circular-Toggler

**Comment by tobehonest** [comment]
1. After having a fixed sidebar, can you remove the menu that is shown at the bottom too , having two menu at the same time looks bad.
Add https://github.com/fossasia/susi.ai/issues/3461 to the fixes too. 
Next time please make sure that no one else is working on the same issue.

**Comment by tobehonest** [comment]
issue

The scroll bar of the skill-cards behind is visible in the sidebar. Can you look into that?

File: src/components/cms/BrowseSkill/SideDrawer.js [hunk 1]
```diff
@@ -13,11 +14,14 @@ const useStyles = makeStyles({
   },
 });
 
-const Button = styled.button`
-  float: right;
+const Button = styled(Fab)`
+  position: fixed;
+  bottom: 20px;
```
Comment (tobehonest) [inline src/components/cms/BrowseSkill/SideDrawer.js]: The button is not completely aligned with chat bubble and scrolltop. The difference was clearly visible when i checked using my UE.
alignemnt

**Comment by MuhammadWasif** [comment]
Please squash your commits into one commit. There should be one feature at a time.
Rest looks good to me

**Comment by elit-altum** [comment]
@akshatnitd @MuhammadWasif  Should I remove the skills tab on the main page and let it stay only on the sidebar as @tobehonest  said ?

**Comment by elit-altum** [comment]
> This is a very bad pull request - 62 files have changes, most of them without the need to change them for the issue.
> Many (most?) cases of changes are indentation changes. You should not do this! The worst about such unnecessary changes is, that
> 
> * it destroys the ability to track changes from previous issues
> * it creates conflicts for other pull requests.
> 
> I strongly suggest that you remove _all_ unnecessary changes!

@Orbiter sorry about this. I am still new at squashing commits, so while squashing my commits, the previous commits also came out as I had merged the upstream to update my branch. I'd really appreciate any help to avoid such circumstances in the future. Thank you!

**Comment by snitin315** [comment]
```
git fetch upstream master
git rebase upstream master
```

**Pull request closed without merging**